ModelImporter.h

```diff
@@ -1,13 +1,11 @@
 #pragma once
 
+#include <filesystem>
 #include <optional>
+#include <utility>
 
 struct Mesh;
 struct Chunk;
 
-namespace std::filesystem {
-	class path;
-}
```

Comment:
[N4659 20.5.4.2.1 Namespace std](https://timsong-cpp.github.io/cppwp/n4659/namespace.std), which is the [c++ 17 draft](https://www.open-std.org/jtc1/sc22/wg21/docs/standards), says:

> The behavior of a C++ program is undefined if it adds declarations or definitions to namespace std or to a namespace within namespace std unless otherwise specified. A program may add a template specialization for any standard library template to namespace std only if the declaration depends on a user-defined type and the specialization meets the standard library requirements for the original template and is not explicitly prohibited.

This is an error that occurs due to this:
```
In file included from /usr/i686-w64-mingw32/sys-root/mingw/include/c++/filesystem:48,
                 from /home/maarten/projects/c47edit/ObjModel.cpp:5:
/usr/i686-w64-mingw32/sys-root/mingw/include/c++/bits/fs_fwd.h:318:19: error: reference to 'path' is ambiguous
  318 |   void copy(const path& __from, const path& __to, copy_options __options);
      |                   ^~~~
/usr/i686-w64-mingw32/sys-root/mingw/include/c++/bits/fs_fwd.h:58:9: note: candidates are: 'class std::filesystem::__cxx11::path'
   58 |   class path;
      |         ^~~~
In file included from /home/maarten/projects/c47edit/ObjModel.cpp:1:
/home/maarten/programming/c47edit/ObjModel.h:11:15: note:                 'class std::filesystem::path'
   11 |         class path;
      |               ^~~~
```

Reply:
If you want to speed up the inclusion, I'd suggest to look into precompiled headers instead.
This will give compile time speed-ups for all compile units.

Reply:
You're right, then indeed we can just include <filesystem> in the header.